tests/README.md

```diff
@@ -31,3 +31,25 @@ Installs all the required mvn dependencies.
 ### `mvn test`
 
 Runs the tests on the specified browser.
+
+## Running the Cucumber Tests with tags
+Add values to the keys in config.properties file
+Update tags value in 'RunCucumberTest' class
+
+In the 'tests' directory, you can run:
+
+### `mvn test -Dcucumber.options="--tags '@tagname'"`
```

Comment:
I think this should be ```bash BLAH ``` for a bash command

Reply:
triple back tick COMMAND triple back tick...sorry didn't anticipate formatting there

Reply:
Fixed. Thanks!